apps/api/src/domain/models/user-management/group/entities/invariant-validation/validateCoscradUserGroup.ts

```diff
@@ -1,6 +1,5 @@
 import { ValidationResult } from '../../../../../../lib/errors/types/ValidationResult';
 import { Valid } from '../../../../../domainModelValidators/Valid';
 
-export default (_: unknown): ValidationResult => {
-    return Valid;
-};
+// There are no complex invariant validation rules for this model
+export default (_: unknown): ValidationResult => Valid;
```

Comment:
Nice. Let's try to get rid of the need for an external function by refactoring the `domain model factories`.